pkgs/by-name/ne/nemu/package.nix

```
@@ -0,0 +1,64 @@
+{ cmake
+, dbus
+, fetchFromGitHub
+, gettext
+, graphviz
+, json_c
+, libarchive
+, libusb1
+, libxml2
+, ncurses
+, ninja
+, openssl
+, pkg-config
+, sqlite
+, stdenv
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nemu";
+  version = "3.3.1";
+
+  src = fetchFromGitHub {
+    owner = "nemuTUI";
+    repo = "nemu";
+    rev = "v${version}";
+    hash = "sha256-6WzqBkspKKs1e8kg1i71ntZHa78s5pJ1u02mXvzpiEc=";
+  };
+
+  cmakeFlags = [
+    "-DNM_WITH_DBUS=ON"
+    "-DNM_WITH_NETWORK_MAP=ON"
+    "-DNM_WITH_REMOTE=ON"
+    "-DNM_WITH_USB=ON"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    dbus
+    gettext
+    graphviz
+    json_c
+    libarchive
+    libusb1
+    libxml2
+    ncurses
+    openssl
+    sqlite
+  ];
+
+  meta = {
+    description = "Ncurses UI for QEMU";
+    homepage = "https://github.com/nemuTUI/nemu";
+    license = lib.licenses.bsd2;
+    mainProgram = "nemu";
+    maintainers = with lib.maintainers; [ msanft ];
+    platforms = lib.platforms.linux;
```

```suggestion
    platforms = lib.platforms.unix;
```
According to [this section](https://github.com/nemuTUI/nemu?tab=readme-ov-file#environment-requirements) in the README it can be used on Linux/Mac/FreeBSD. You can set it to `lib.platforms.unix`.